### PR TITLE
Add LLM Agent notebook

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
hi @eaidova any idea on how to add meta data ? thanks

**Comment by OpenVINO-dev-contest** [comment]
@ryanloney FYR

File: notebooks/llm-agent/llm-agent.ipynb [hunk 1]
```diff
@@ -0,0 +1,717 @@
+{
```
Comment (eaidova) [inline notebooks/llm-agent/llm-agent.ipynb:1]: <p>please provide info about which model used as example here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (eaidova) [inline notebooks/llm-agent/llm-agent.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        !optimum-cli export openvino --model {model_id} --weight-format int4 {model_path}
</pre><p>need to add $ before {} for resolving variables in cli</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (eaidova) [inline notebooks/llm-agent/llm-agent.ipynb:1]: <p>please add link on provided optimizations in docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (eaidova) [inline notebooks/llm-agent/llm-agent.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        ["Can you introduce the latest generation of iPhone to me ?"],
</pre><p>not sure that we can refeence IPhone in notebook. Maybe better to use another example, e.g. what is Eiffel tower tall?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

File: notebooks/llm-agent-langchain/llm-agent-langchain.ipynb [hunk 2]
```diff
@@ -0,0 +1,719 @@
+{
```
[the remark this answers is not in the document]
Comment (OpenVINO-dev-contest) [reply notebooks/llm-agent-langchain/llm-agent-langchain.ipynb:1]: done

File: notebooks/llm-agent/llm-agent.ipynb (hunk 1 above)
Comment (OpenVINO-dev-contest) [reply notebooks/llm-agent/llm-agent.ipynb]: done

Comment (OpenVINO-dev-contest) [reply notebooks/llm-agent/llm-agent.ipynb]: done

Comment (OpenVINO-dev-contest) [reply notebooks/llm-agent/llm-agent.ipynb]: done

Comment (eaidova) [reply notebooks/llm-agent/llm-agent.ipynb]: I mean this doc https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide/llm-inference-hf.html#enabling-openvino-runtime-optimizations

**Review by eaidova — APPROVED** [review]

**Pull request merged**